app/helpers/mb_form_builder.rb

```diff
@@ -179,11 +179,11 @@ def mb_checkbox(method, label_text, options = {})
 
   def label_contents(label_text, notes, optional = false)
     notes = Array(notes)
-    if optional
-      optional_text = "<span class='form-card__optional'>(optional)</span>"
-    else
-      optional_text = ""
-    end
+    optional_text = if optional
+                      "<span class='form-card__optional'>(optional)</span>"
+                    else
+                      ""
+                    end
```

Comment:
I personally prefer the previous version of this, but I suppose that's another PR.

Reply:
🤗